Practise11_05/Task2/Plants.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Practise11_05.Task2
+{
+    class Plant
+    {
+        public string name;
+
+        public double color;
+
+        public double height;
+
+        public double Water { get; set; }
```

Comment:
I think we should limit access for setters some how. For example make setter private.